### PR TITLE
added ping module

### PR DESCRIPTION
**Pull request by sasek333**
requires os's ping access

### CODE REVIEW

**Comment by swapagarwal** [comment]
Can we leverage an API call for this?

**Comment by sasek333** [comment]
You mean like pyping? It requires root on linux machines. I'll check it in a week because from today it's vacation time :)

File: modules/src/ping.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    host = entities['host'][0]['value']
+
+    if not host:
+        output['success'] = False
+        output['output'] = "Please specify a host"
+
+        return output
+
+    ping_str = "-n 1" if platform.system().lower() == "windows" else "-c 1"
```
Comment (swapagarwal) [inline modules/src/ping.py]: this `platform` thing is bugging me!

**Comment by swapagarwal** [comment]
Keeping in mind that it should be cross-platform compatible, we have two options: either use a python library (sockets / requests) to check website availability, or use an external service like [`Is It Up`](https://isitup.org/github.com).
Hit me up after your vacations. Enjoy! :smile:

**Comment by swapagarwal** [comment]
Hey @sasek333, will you be willing to complete this? :smile:

**Comment by shanmukhateja** [comment]
Actually @swapagarwal something's been bothering me for a while.  The only use of ping module is to check if the internet connection is working or not. 

And this condition is already satisfied when we are using messenger right? because it's only then we can use messenger right?

Am I missing something or is there something else to it?

**Comment by swapagarwal** [comment]
@shanmukhateja Well, this would check whether a website can be reached or not, say is github.com down or not?

**Comment by shanmukhateja** [comment]
Oh OK so it was meant for another website's working state. Good :)

**Comment by VarunWachaspati** [comment]
@swapagarwal :If the intention is just to check whether a website is up or down, then we can request for the page from our host server if it returns a response then it is up right else if it times out for say continuously three packets then it must be down.

This way I guess cross compatibility will be maintained and purpose will also be served. If any other use case comes to your mind for which the above does not cover let me know.

**Comment by swapagarwal** [comment]
@VarunWachaspati Yes, we can implement this on our server itself, but that adds to the maintenance cost!

**Comment by Wert1996** [comment]
May I take this up? How about implementing this using isitup.org to check whether a website is down or not?

**Comment by swapagarwal** [comment]
Added in #197

**Pull request closed without merging**